modules/depth_prediction/layers.py

```diff
@@ -0,0 +1,269 @@
+# Copyright Niantic 2019. Patent Pending. All rights reserved.
```

Comment:
is it necessary to include all the Monodepth2 training source code in this patch? can you just add a pre-trained model in this patch?

Reply:
I have prepared the pre trained model, and the user can download the model to the specified folder and reasoning directly. The inference part does not need this part of code.

This part of the code about the network is introduced for the model transformation part. Considering that if multiple .PTH file format models trained by users want to be converted to IR format, the network needs to be loaded first for analysis and integrated into an intermediate file in .Onnx format.

Reply:
You can add comments in README.md to introduce how to train a Monodepth2 model from the original github repo.
The patch should only include inference part, add some introduction of the external model being used.
Please refer to an example of openvino model zoo
https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/fcrn-dp-nyu-depth-v2-tf

Reply:
Also please remove all python cache files
modules/depth_prediction/__pycache__/layers.cpython-36.pyc
modules/depth_prediction/__pycache__/utils.cpython-36.pyc